frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/MembersPagePage/components/AddMemberModal/AddMemberModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useCallback, useState } from "react";
+
+import { IUser } from "interfaces/user";
+import { INewMembersBody } from "interfaces/team";
+import endpoints from "kolide/endpoints";
+import Modal from "components/modals/Modal";
+import Button from "components/buttons/Button";
+import AutocompleteDropdown from "components/forms/fields/AutocompleteDropdown";
+
+const baseClass = "add-member-modal";
+
+interface IAddMemberModal {
+  onCancel: () => void;
+  onSubmit: (userIds: INewMembersBody) => void;
+}
+
+const AddMemberModal = (props: IAddMemberModal): JSX.Element => {
+  const { onCancel, onSubmit } = props;
+
+  const [selectedMembers, setSelectedMembers] = useState([]);
+
+  const onChangeDropdown = useCallback((values) => {
+    setSelectedMembers(values);
+  }, []);
+
+  const onFormSubmit = useCallback(() => {
+    const userIds = selectedMembers.map((member: IUser) => {
+      return { id: member.id, role: "observer" };
```

Comment:
Does `"observer"` need to come from some variable?

Reply:
not in this case. new members added are always added with observer role.